pixiedust/display/chart/renderers/baseChartDisplay.py

```diff
@@ -114,6 +114,7 @@ def wrapper(cls, *args, **kwargs):
 class BaseChartDisplay(with_metaclass(ABCMeta, ChartDisplay)):
 
     commonOptions = co
+    extraFields = []
```

Comment:
@glynnbird I'm afraid we still have the same issue. As declared extraFields is a static field that once modified will apply to all subclasses.
You should leave the default body of getExtraFields function to return [] and have it overridden in the concrete class to return the non-numeric fields

Reply:
Gotcha. New commit on its way